main.go

```diff
@@ -19,12 +19,14 @@ package main
 
 import (
```

Comment:
Just to make it visible here: Let's also document this in the v1.2-v1.3 migration guide and link the current PR as a reference PR

Reply:
Does this PR require any action from providers?

Reply:
If so we should follow the new process since beta is already out:

> All changes that impact providers' adoption of the new release should be announced in the provider updates section of the office hours meeting notes and approved in the PR or issue by both approvers and key affected providers

https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/developer/release-cycle.md#release-cycle

Reply:
In my opinion it doesn't **require** any action. We would add a recommendation to the migration guide as we usually do.

But the migration guide itself is not binding and part of the contract. Some providers follow it, some don't (at least that's my impression from previous CAPI bumps).

Reply:
From the wording of the guideline pointed out by Cecile above, this is not a change that needs to be adopted by the providers to adopt this CAPI release. This is more of a good to make change(**klog logging change**) rather than a must make change(**like SSA**).
My vote would be include these in the migration guide with an explicit callout saying that this is not required to be implemented/used by providers, but it is a good to be included for the sake of consistency amongst CAPI and providers.

Reply:
Sounds like a good trade-off to me

Reply:
@CecileRobertMichon Move the provider-relevant part to https://github.com/kubernetes-sigs/cluster-api/pull/7511